docs/integrations/linode/gslb.mdx

```diff
@@ -0,0 +1,322 @@
+
+Create a Weighted Backend at the [`/backends/weighted` endpoint](https://ngrok.com/docs/api/resources/weighted-backends/) using your Tunnel Group backend `id`s exported from the previous step. The `curl` command below creates equal weigting for your backends, but you can use integers between `0-10000` to configure precise proportional weights.
+
+```bash
+curl \
+	-X POST https://api.ngrok.com/backends/weighted \
+	-H "Authorization: Bearer ${NGROK_API_KEY}" \
+	-H "Content-Type: application/json" \
+	-H "Ngrok-Version: 2" \
+	-d '{"backends":{"'${BACKEND_ID_01}'":1,"'${BACKEND_ID_02}'":1,"'${BACKEND_ID_03}'":1}}'
+```
+
+Export the `id` of your new Weighted Backend.
+
+```bash
+export WEIGHTED_BACKEND_ID=bkdwd_...
+```
+
+You can now create an catch-all Edge Route at `/` using your Weighted Backend using the [`/edges/https/{EDGE_ID}/routes` endpoint](https://ngrok.com/docs/api/resources/edges-https-routes/).
+
+```bash
+curl \
+  -X POST "https://api.ngrok.com/edges/https/${EDGE_ID}/routes" \
+  -H "Authorization: Bearer ${NGROK_API_KEY}" \
+  -H "Content-Type: application/json" \
+  -H "Ngrok-Version: 2" \
+  -d '{"match":"/","match_type":"path_prefix","backend":{"enabled":true,"backend_id":"'${WEIGHTED_BACKEND_ID}'"}}' \
+```
+
+You now have an active domain, Edge, Tunnel Group backends, a Weighted Backend, and a route. If you open a new tab and navigate to `https://<YOUR_NGROK_DOMAIN>` right now, you'll see an error message from ngrok. You've successfully reserved a domain and created an edge, but don't yet have ngrok agents or active tunnels.
+
+![An example of an ngrok domain and Edge with no matching tunnels](img/edge-no-tunnels.png)
+
+You can also navigate to the [Edges dashboard](https://dashboard.ngrok.com/cloud-edge/edges) to see your Edge configuration, on the domain you reserved, with no online tunnels.
+
+![alt text](img/edge-dashboard-no-tunnels.png)
+
+## **Step 5**: Install the ngrok agent and an example workload on each VM {#install-ngrok-agent}
+
+To get some tunnels online and quickly see how ngrok's GSLB works, you'll use an [example API deployment](https://github.com/joelhans/ngrok-vps-gslb-demo). This demo deployment has four parts:
+
+1. A straightforward Go-based API with a single endpoint at `/api`, which returns a randomly generated UUID and the machine's `hostname`, which should reflect the regions of your VMs.
+2. A `Dockerfile` for containerizing said Go-based API.
+3. A `docker-compose.yml` file for starting the API container and a containerized edition of the ngrok agent on the same network.
+4. A `ngrok.yml` [agent configuration file](/docs/agent/config.mdx) to connect the agent to your ngrok account and one of the Tunnel Group backends you previously created.
+
+Repeat the following steps on **each** VM:
+
+1. Clone the demo repository.
+
+   ```bash
+   git clone https://github.com/joelhans/ngrok-vps-gslb-demo
```

Comment:
It should! The way we've done this in the past is to have someone in ngrok fork my repository so that it points to something like `https://github.com/ngrok/ngrok-vps-gslb-demo` instead.

I don't know if you have privileges to do that, but I'm more than happy to switch the URL when it's done.